src/ripple/app/ledger/LedgerConsensus.cpp

```diff
@@ -66,6 +66,22 @@ class LedgerConsensusImp
     , public std::enable_shared_from_this <LedgerConsensusImp>
     , public CountedObject <LedgerConsensusImp>
 {
+private:
+    enum class State
+    {
+        // We haven't closed our ledger yet, but others might have
+        open,
+
+        // Establishing consensus
+        establish,
```

Comment:
This one word stands out as a verb in the infinitive where the others act like adjectives...?

Why not call it `establishing`?